### ISSUE TITLE
This Week at SCRF

### ISSUE THREAD

**Issue by vincenzospaghetti** [smartcontractresearchforum/engagement#153]

**Comment by vincenzospaghetti** [comment]
![Image](https://user-images.githubusercontent.com/93289575/184701950-3b60cbdd-ef90-49d7-a501-808b4f0f5569.png)

**Comment by zubepaul** [comment]
This is a good graphic to start with. Perhaps something that @marvino3 and team can look at once this project moves forward

**Comment by zubepaul** [comment]
This ticket now covers #121 as well

**Comment by zubepaul** [comment]
Issue 121 has a draft of the project plan that should be migrated into this ticket.

**Comment by zubepaul** [comment]
This could potentially be a contributor level project that runs on the forum.

**Comment by 0xAngle** [comment]
[This Week @ SCRF Template.pdf](https://github.com/smartcontractresearchforum/engagement/files/9844893/This.Week.%40.SCRF.Template.pdf)

@marvino3 

thoughts on this for a template for weekly posting in Discord? Paul mentioned doing something different for the forum though

**Comment by marvino3** [comment]
This is awesome! Are we going to be posting these as pdfs @0xAngle?  If so, I'd love to have our designer take a quick pass at this.

**Comment by 0xAngle** [comment]
Yes the intention is to have this be a source of headlines of SCRF happenings to be posted in Discord in the PDF format. I want to be able to add links to the topics if they exist but I'm not sure discord supports that. @marvino3

**Comment by 0xAngle** [comment]
I'd love to have the designed give some input. That notepad graphic is definitely pretty blurry and i see some issues with text and whatnot that need some help

**Comment by 0xAngle** [comment]
So I think the current state of this is:

- We are working on locating a permanent home for this document to be so it can be visible on the forum and easily forwarded (and visible in preview) to discord

- I am searching for a way to integrate clickable links into the document that can direct people to the specific points. This will point them toward the place of discussion if they are so interested in any given topic

- I have the ideas for content indicated, but reiterated they are: Updates from Verticals, Trending content on the forum, Recap happenings of the week (guild meetings, community call, coffee house, SCRF-wide announcements), and a Preview of the next week

- I am working with brian on a pipeline for information that will help populate the Updates from Verticals section: https://github.com/smartcontractresearchforum/IT/issues/132

**Comment by marvino3** [comment]
Update from design: @MattDesigner is working on SCRF Spotlight imagery @zubepaul @0xAngle

**Comment by zubepaul** [comment]
Next steps while waiting on design @0xAngle 
- Process document about how this is made each time
- Produce a draft template for what it looks like

**Comment by 0xAngle** [comment]
https://docs.google.com/document/d/1EWSm9cejkhytDPN0haE1QDIS8HmyUiS6vTUfHBOk-gc/edit?usp=sharing

A draft template would require the design, no?

**Comment by zubepaul** [comment]
It will require some of the design, but we also want to make sure that the substance makes sense

**Comment by SirAh0** [comment]
Friendly reminder that the Announcements Funnel is up and running. 

Still working out the process as we go but any kind of announcement should be routed through here to make sure it gets the SCRF treatment.

Announcement requests can be made here:
📣 - Announcement Request
https://github.com/smartcontractresearchforum/engagement/issues/new/choose

**Comment by 0xAngle** [comment]
> Update from design: @MattDesigner is working on SCRF Spotlight imagery @zubepaul @0xAngle

Hey there! Just want to check on the status of this

**Comment by MattDesigner** [comment]
Hi! I think the initial designs are with still with @marvino3 for review?

**Comment by marvino3** [comment]
Yes, thank you for the push. Making a few revisions before sharing.

**Comment by MattDesigner** [comment]
Design amends have been sent back to Maria for approval before sharing.

**Comment by marvino3** [comment]
Thanks Matt. @0xAngle @zubepaul please find the draft here: https://drive.google.com/file/d/1588FjBHjK8grNnBt3gPndQ3RlBvoCMVs/view?usp=share_link

Lmk if anyone needs permissions.

**Comment by 0xAngle** [comment]
I think this is super great! Is there a way I can get an editable version of this?

Thanks

**Comment by zubepaul** [comment]
@marvino3 @0xAngle 
We are still thinking this will go on the /docs repo and then linked to/from, correct?

**Comment by marvino3** [comment]
@0xAngle awesome, it is editable for me if you open it w Acrobat Pro. Was there another format you were hoping for?

@zubepaul I think it does make sense to add those to /docs and probably promo/ refer to it in chat and other spots.

**Comment by zubepaul** [comment]
As a recap of where we are on this ticket and process, this is my current understanding that we would like to go forward with:
- We're renaming this SCRF Spotlight
- We have a logo/visual
- We're going to publish these on the /docs repo so that they can be shared via socials and easily in chat
- @0xAngle is primarily responsible for gathering these up, but other verticals/projects should have a way of getting these types of materials to him.
- We need a process to publish. Is it reasonable that the draft form of SCRF Spotlight goes to the SCRF Messaging board and works through @isss111 process? If so, what is the time needed to go through that pipeline? If that is not a good process, what would a good alternative be? Thoughts @marvino3

**Comment by marvino3** [comment]
@zubepaul yes, I think it makes sense to add these to the Messaging board since we intend to publish them on /docs. For the time needed, I'd say about a week, conservatively. Lmk if you think that timing works @isss111
